ecosystem/typescript/sdk/src/transaction_builder/builder.ts

```diff
@@ -68,16 +67,16 @@ export class TransactionBuilder<F extends SigningFn> {
   static getSigningMessage(rawTxn: AnyRawTransaction): SigningMessage {
     const hash = SHA3.sha3_256.create();
     if (rawTxn instanceof RawTransaction) {
-      hash.update(Buffer.from(RAW_TRANSACTION_SALT));
+      hash.update(RAW_TRANSACTION_SALT);
     } else if (rawTxn instanceof MultiAgentRawTransaction) {
-      hash.update(Buffer.from(RAW_TRANSACTION_WITH_DATA_SALT));
+      hash.update(RAW_TRANSACTION_WITH_DATA_SALT);
     } else {
       throw new Error("Unknown transaction type.");
     }
 
     const prefix = new Uint8Array(hash.arrayBuffer());
 
-    return Buffer.from([...prefix, ...bcsToBytes(rawTxn)]);
+    return Uint8Array.from([...prefix, ...bcsToBytes(rawTxn)]);
```

Comment:
This PR needs a rebase. Thought I have get rid of these spread operators for Uint8Array